packages/e2e-tests/scripts/record-playwright.ts

```diff
@@ -41,9 +41,7 @@ export async function recordPlaywright(
   });
   const page = await context.newPage();
   try {
-    return await script(page as any, expect);
```

Comment:
for context - there are 2 timeouts at play here, 30s timeout in `defaultPlaywrightScript` (within `waitUntilMessage`) and the 60s timeout based on the `CONFIG.recordingTimeout` value. When the latter happens it already throws nicely but when the former one gets thrown... this was caught here and swallowed

Reply:
I _think_ the original idea here was to allow some tests to re-record and upload even if others failed (but the fact that the script (a) never re-threw an error later and (b) actually continued to upload a potentially failed recording was just _broken__